src/OrleansAzureUtils/Providers/Storage/AzureBlobStorage.cs

```diff
@@ -91,14 +91,21 @@ public class AzureBlobStorage : IStorageProvider
 
         IEnumerable<string> FormatPropertyMessage(IProviderConfiguration config)
         {
-            foreach (var property in new string[] { ContainerNamePropertyName, "SerializeTypeNames", "PreserveReferencesHandling", "UseFullAssemblyNames", "IndentJSON" })
+            var properties = new[]
+            {
+                ContainerNamePropertyName,
+                "SerializeTypeNames",
+                "PreserveReferencesHandling",
+                OrleansJsonSerializer.UseFullAssemblyNamesProperty,
```

Comment:
They seem to be bogus, I was considering removing them but wasn't 100% certain. I found no occurrences of those strings anywhere in our solution.

I'll nix them.